tests/test_mock_controller.py

```diff
@@ -1182,3 +1198,26 @@ async def test_status(self) -> None:
         thcs_status = self.data[mtdome.LlcName.THCS.value]
         assert thcs_status["status"]["status"] == mtdome.LlcMotionState.CLOSED.name
         assert thcs_status["temperature"] == [0.0] * mtdome.mock_llc.NUM_THERMO_SENSORS
+
+    async def test_slow_network(self) -> None:
+        self.mock_ctrl.slow_network = True
+        await self.write(command="statusAMCS", parameters={})
+        with pytest.raises(asyncio.exceptions.TimeoutError):
+            # The default timeout should time out because of the slow network.
+            await self.read()
+        # Waiting longer should eventually result in a successful read.
+        self.data = await self.read(timeout=SLOW_NETWORK_TIMEOUT)
+        amcs_status = self.data[mtdome.LlcName.AMCS.value]
+        assert amcs_status["status"]["status"] == mtdome.LlcMotionState.PARKED.name
+        assert amcs_status["positionActual"] == 0
+
+    async def test_network_interruption(self) -> None:
```

Comment:
Again, a brief doc string would be helpful.

Reply:
Done